test/mocks/base-mock-eos-report.js

```diff
@@ -69,6 +72,65 @@ exports.genericCustomizedEosReportRequest = {
     infoForRmRc: genericCustomizedEosReport.infoForRmRc,
 };
 
+// eslint-disable-next-line require-jsdoc
+const genericCustomTimeEosReport = (time, information) => ({
```

Comment:
What is the purpose of these mocks?

Reply:
To be able to properly test the autofill feature from 8 hours ago